blocks/library/block/edit-panel/index.js

```diff
@@ -10,66 +11,97 @@ import { keycodes } from '@wordpress/utils';
  */
 import './style.scss';
 
+/**
+ * Module constants
+ */
 const { ESCAPE } = keycodes;
 
-function ReusableBlockEditPanel( props ) {
-	const { isEditing, title, isSaving, onEdit, onChangeTitle, onSave, onCancel } = props;
+class ReusableBlockEditPanel extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.bindTitleRef = this.bindTitleRef.bind( this );
+		this.handleFormSubmit = this.handleFormSubmit.bind( this );
+		this.handleTitleChange = this.handleTitleChange.bind( this );
+		this.handleTitleKeyDown = this.handleTitleKeyDown.bind( this );
+	}
+
+	bindTitleRef( ref ) {
+		this.titleRef = ref;
+	}
+
+	componentDidMount() {
+		if ( this.props.isEditing ) {
+			this.titleRef.select();
+		}
+	}
 
-	return [
-		( ! isEditing && ! isSaving ) && (
-			<div key="view" className="reusable-block-edit-panel">
-				<span className="reusable-block-edit-panel__info">
-					<b>{ title }</b>
```

Comment:
Of course not relevant to this pull request, but was `<b>` chosen for a particular reason?

>Do not confuse the <b> element with the <strong>, <em>, or <mark> elements. The <strong> element represents text of certain importance, <em> puts some emphasis on the text and the <mark> element represents text of certain relevance. The <b> element doesn't convey such special semantic information; use it only when no others fit.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b

Further, do we need _both_ the `span` and the `b`, or could we collapse to eliminate one or the other (probably just replacing `span` with desired `inline` tag).

Reply:
Eliminated the `<span>` in 3b92ad56e11687850d0efb378bdda1437e0be90c. I like the `<b>` in lieu of a `<span>` since it denotes that there's something special about this text to the developer who is reading the JSX.